akka-actor/src/main/scala/akka/dispatch/Future.scala

```diff
@@ -75,32 +72,6 @@ object ExecutionContexts {
    */
   def global(): ExecutionContextExecutor = ExecutionContext.global
 
-  /**
-   * INTERNAL API
-   *
-   * WARNING: Not A General Purpose ExecutionContext!
-   *
-   * This is an execution context which runs everything on the calling thread.
-   * It is very useful for actions which are known to be non-blocking and
-   * non-throwing in order to save a round-trip to the thread pool.
-   *
-   * Once Scala 2.12 is no longer supported this can be dropped in favour of directly using `ExecutionContext.parasitic`
-   */
-  @InternalStableApi
-  private[akka] val parasitic: ExecutionContext = SameThreadExecutionContext()
```

Comment:
That this is `@InternalStableApi` means that we use it all over other Akka modules, just so you know they will no longer compile without changes

Reply:
So we better not remove it? I was assuming we anyway have to make changes after the first M1 will be out.

Reply:
I think it is fine, just wanted to make you aware if you were not

Reply:
I think it's only Akka HTTP that builds from latest, so would be good to follow up there right away (to avoid ci email spamming)

Reply:
https://github.com/akka/akka-http/pull/4422